files/ru/learn/tools_and_testing/index.md

```diff
@@ -20,26 +20,35 @@ tags:
   - user testing
 translation_of: Learn/Tools_and_testing
 ---
```

Comment:
```suggestion
---

```

Reply:
@yin1999 thanks for suggested change. whom may i ask to approve changes ?